README.md

```diff
@@ -1,308 +1,60 @@
+MAC Address from the pool.
 
-You can opt-in kubemacpool MAC assignment on your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocate` - to opt-in MAC assignment for Pods in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt-in MAC assignment for VMs in your namespace
+For example, a simple VirtualMachine can be created by issuing the following commands:
 
-To disable kubemacpool MAC assignment on a specific namespace:
 ```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
-namespace/example-namespace labeled
-```
-
-To re-enable kubemacpool MAC assignment in a namespace:
-```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io=allocate mutatevirtualmachines.kubemacpool.io=allocate
-namespace/example-namespace labeled
-```
-
-**note:** If a VMI is created directly and not through a VM, then it is handled in kubemacpool by the pod handler.
-
-### Check deployment
-
-Configmap:
-```bash
-[root]# kubectl -n kubemacpool-system describe configmaps
-
-Name:         kubemacpool-mac-range-config
-Namespace:    kubemacpool-system
-Data
-====
-RANGE_END:
-----
-FD:FF:FF:FF:FF:FE
-
-RANGE_START:
-----
-02:00:00:00:00:11
-```
-
-pods:
-```bash
-kubectl -n kubemacpool-system get po
-NAME                                                  READY   STATUS    RESTARTS   AGE
-kubemacpool-mac-controller-manager-6894f7785d-t6hf4   1/1     Running   0          107s
-```
-
-### Example
-
-Create a network-attachment-definition:
-
-The 'NetworkAttachmentDefinition' is used to setup the network attachment, i.e. secondary interface for the pod.
-This is follows the [Kubernetes Network Custom Resource Definition De-facto Standard](https://docs.google.com/document/d/1Ny03h6IDVy_e_vmElOqR7UdTPAG_RNydhVE1Kx54kFQ/edit)
-to provide a standardized method by which to specify the configurations for additional network interfaces.
-This standard is put forward by the Kubernetes [Network Plumbing Working Group](https://docs.google.com/document/d/1oE93V3SgOGWJ4O1zeD1UmpeToa0ZiiO6LqRAmZBPFWM/edit).
-
-```yaml
-apiVersion: "k8s.cni.cncf.io/v1"
-kind: NetworkAttachmentDefinition
-metadata:
-  name: ovs-conf
-  annotations:
-    k8s.v1.cni.cncf.io/resourceName: ovs-cni.network.kubevirt.io/br1
-spec:
-  config: '{
-      "cniVersion": "0.3.1",
-      "name": "ovs-conf",
-      "plugins" : [
-        {
-          "type": "ovs",
-          "bridge": "br1",
-          "vlan": 100
-        },
-        {
-          "type": "tuning"
-        }
-      ]
-    }'
+wget https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/doc/simple-vm.yaml
+kubectl apply -f simple-vm.yaml
 ```
 
-This example used [ovs-cni](https://github.com/kubevirt/ovs-cni/).
+After VirtualMachine is created, you can check the MAC Address under interfaces section:
```

Comment:
Could we change this into a copy-paste example?

```
kubectl get vm simplevm
```

```yaml
the example you already have here
```

Reply:
Done